src/isValidQuery.js

```diff
@@ -0,0 +1,6 @@
+const minMax = new RegExp(/(\()(?:[a-z][a-z]+)(-)((?:[a-z][a-z]+))?(-)?((?:[a-z][a-z]+))(:)(\s?)(\d+)((?:[a-z][a-z]+))\)/)
```

Comment:
@kjaenicke It's tricky though, browsers have a lot of non-standard media queries but it could be done. You should open issues on css-mediaquery to do validation, and we can use that to validate when that is done.